packages/graphback-runtime-mongodb/src/OffixMongoDBDataProvider.ts

```diff
@@ -10,17 +10,50 @@ import { MongoDBDataProvider } from './MongoDBDataProvider';
  * https://offix.dev/docs/conflict-server#structure-of-the-conflict-error
  */
 export class OffixMongoDBDataProvider<Type = any, GraphbackContext = any> extends MongoDBDataProvider<Type, GraphbackContext> {
+  protected updatedAtField: string;
+  protected createdAtField: string;
 
   public constructor(baseType: GraphQLObjectType, client: any) {
     super(baseType, client);
+    this.createdAtField = undefined;
+    this.updatedAtField = undefined;
+    Object.keys(baseType.getFields()).forEach((k: string) => {
```

Comment:
This can be implemented in MongoDBDataProvider instead so that it can be use without having to install Offix.

You could also abstract the directive parsing out of the data provider so it can be reused across different providers (since there is nothing specific about Mongo here yet).